src/test/java/pages/LoginPage.java

```diff
@@ -0,0 +1,38 @@
+package pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+
+public class LoginPage extends BasePage{
+
+    public LoginPage(WebDriver givenDriver) {
+        super(givenDriver);
+    }
+
+
+    //Web Elements;
+    By emailField = By.cssSelector("input[type='email']");
+    By passwordField = By.cssSelector("input[type='password']");
+    By loginBtn = By.cssSelector("button[type='submit']");
+
+
+    // Helper Method;
+
+   /* public void provideEmail(String email){
+        findElement(emailField).sendKeys(email);
+
+    }
+    public void providePassword(String password){
+        findElement(passwordField).sendKeys(password);
+    }
+    public void clickSubmit(){
+        findElement(loginBtn).click();
+    }*/
```

Comment:
I recommend not using code comments: for, example, when you have a code that cannot be understood, then comments are not the solution. Try to rewrite the code or rename fields and other elements in a way that the reader will understand the action that you are doing there.